config/default.schema.json

```diff
@@ -1660,6 +1660,11 @@
 			"type": "boolean",
 			"default": false,
 			"description": "Enables the AI Tutor"
+		},
+		"FEATURE_ROOMS_ENABLED": {
+			"type": "boolean",
+			"default": "false",
+			"description": "Enables the rooms feature"
 		}
 	},
 	"required": []
```

Comment:
if you want this to be enabled on local and old e2e, then you can add to true on development.json and test.json

Reply:
oh! good point! not sure about the e2e currently, but on local would be nice